test/unit/metrics/test_metrics_engine.py

```diff
@@ -43,6 +47,35 @@ def test_smoke(self, transform_y_a, transform_y_p, transform_gid):
         assert metrics.difference_from_summary(result) == 1
         assert metrics.ratio_from_summary(result) == pytest.approx(0.6666666667)
 
+    @pytest.mark.parametrize("transform_gid", conversions_for_1d)
+    @pytest.mark.parametrize("transform_y_p", conversions_for_1d)
+    @pytest.mark.parametrize("transform_y_a", conversions_for_1d)
+    def test_smoke_extra_arg(self, transform_y_a, transform_y_p, transform_gid):
+        y_a = transform_y_a([0, 0, 1, 1, 0, 1, 1, 1])
```

Comment:
I'm guessing `y_p` is predicted `y`, but what does `y_a`  stand for?

Reply:
You're asking me to remember something from six or more months ago.... I suspect I was thinking of 'actual'.

Reply:
That’s a great indication that the abbreviation isn’t working.